src/chain_spec.rs

```diff
@@ -941,6 +960,19 @@ fn altair_genesis(
 			candidacy_bond: 1 * AIR,
 			..Default::default()
 		},
+		block_rewards: altair_runtime::BlockRewardsConfig {
+			collators: initial_authorities
+				.iter()
+				.cloned()
+				.map(|(acc, _)| acc)
+				.collect(),
+			collator_reward: 98_630 * MILLI_AIR,
+			total_reward: 98_630 * MILLI_AIR * 100,
+		},
+		block_rewards_base: altair_runtime::BlockRewardsBaseConfig {
+			currency_id: CurrencyId::Native,
+			amount: 1 * MICRO_AIR,
```

Comment:
Is this correct that the reward for a block  is 1 _micro_ AIR?

Reply:
That's not the block reward but the funding of the `pallet_rewards::Instance` sovereign account which holds all the rewards until they are claimed. As long as this account does not have the ED, not all rewards can be claimed at the same time as it would kill this account.

Reply:
Just out of interest, we could use the ED defined in the runtime instead of hardcoding, right?